sweep/tx_input_set.go

```diff
@@ -361,6 +361,8 @@ func createWalletTxInput(utxo *lnwallet.Utxo) (input.Input, error) {
 		witnessType = input.WitnessKeyHash
 	case lnwallet.NestedWitnessPubKey:
 		witnessType = input.NestedWitnessKeyHash
+	case lnwallet.TaprootPubkey:
```

Comment:
Let's also make the default addresses for the sweeper P2TR as well? 

So we'd change this argument here: https://github.com/lightningnetwork/lnd/blob/master/server.go#L4422-L4424

That'll also indirectly cause all on chain sweeps (force closes, etc) to go into P2TR addresses.

Reply:
We should also be able to change this as well: https://github.com/lightningnetwork/lnd/blob/de6fcf15b6e090613d62fc859f42b418124730a1/peer/brontide.go#L2162-L2175. 

In practice we _should_ always be using the upfront shutdown addr, but this should be safe.

Reply:
We'll also need to update this, so older nodes will accept a P2TR addr as an upfront shutdown script: https://github.com/lightningnetwork/lnd/blob/dceb10144f7188a09f5fcdbe04ce4ef9adf6960a/lnwallet/wallet.go#L2247-L2268

I think this means that we shouldn't yet switch over the default co-op close addr due to this stricter validation.

Reply:
Re the above, here's where we generate our upfront shutdown addr: https://github.com/lightningnetwork/lnd/blob/262591c3331c2170697fe89b8c49ab857db0adf7/funding/manager.go#L1363-L1375

Reply:
Some other areas that are safe to change:
  * https://github.com/lightningnetwork/lnd/blob/bd5e29ec83186f00e2975af3031b4aedad2bccb4/lnd.go#L460-L461
  * https://github.com/lightningnetwork/lnd/blob/c647d32cf7d3da17e3f94f937e1e5cf6b29e2b2f/rpcserver.go#L1328-L1330
  * https://github.com/lightningnetwork/lnd/blob/dceb10144f7188a09f5fcdbe04ce4ef9adf6960a/lnwallet/wallet.go#L765-L766